examples/browser-js/build.gradle.kts

```diff
@@ -10,6 +10,6 @@ subprojects {
         if (System.getenv("CI") == "true") {
             mavenLocal()
         }
-        jcenter()
+        mavenCentral()
```

Comment:
We can only update these, when `pbandk` is actually published to Maven Central.
So this might break the build of this PR.